contracts/FinancialManagement.sol

```
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.23;
+
+contract UserManagement {
+    address public admin;
+    address public manager;
+
+    constructor() {
+        admin = msg.sender;
+    }
+
+    modifier onlyAdmin {
+        require(msg.sender == admin);
```

We should return a message error :v

---

thanks for the suggestion! Updated